raft/src/test/java/org/apache/kafka/raft/MockLogTest.java

```diff
@@ -75,56 +75,6 @@ public void testTopicId() {
         assertEquals(topicId, log.topicId());
     }
 
-    @Test
-    public void testAppendAsLeaderHelper() {
```

Comment:
Why remove the test?

Reply:
I actually, move this to `testAppendAsLeader`. The two tests were testing also the same things. I just merged them so that one test now covers all of the cases that didn't overlap.